conda-forge.yml

```diff
@@ -4,7 +4,7 @@ azure:
     swapfile_size: 10GiB
 bot:
   abi_migration_branches:
-    - 11.8
+    - '11.8'
```

Comment:
Am curious, why was this not read as a string before?

Though the bot was previously reading this ok

Reply:
The context is at from https://github.com/conda-forge/conda-smithy/pull/1756#discussion_r1516567012

Reply:
Sorry, but I was under the impression that all values in `conda-forge.yml` are strings (ex: we have `10GiB` above), so adding quotations or not does not make differences. Since when was this changed? I've read the linked issue but it was unclear to me.

Reply:
YAML parsers try to interpret some values as floats. The “GiB” part of that value makes it parse as a string. This was a big problem when Python 3.10 came out because many YAML files had `- 3.10` which was interpreted as 3.1.

This fix seems reasonable to me.

Reply:
I see, thanks for letting me know the parser quirks...